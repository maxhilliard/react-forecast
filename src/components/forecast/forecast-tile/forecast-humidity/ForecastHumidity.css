.forecastHumidity {
    font-size: 12px;
    composes: forecastHumidity from '../ForecastTile.css';
    margin: 0;
    text-align: left;
}