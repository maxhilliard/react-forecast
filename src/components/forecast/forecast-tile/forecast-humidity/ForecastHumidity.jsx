import React from 'react';
import PropTypes from 'prop-types';

import styles from './ForecastHumidity.css';

const ForecastHumidity = ({ humidity }) => (
    <p className={styles.forecastHumidity}>
        Humidity:
        {humidity}
        %
    </p>
);

ForecastHumidity.propTypes = {
    humidity: PropTypes.number.isRequired,
};

export default ForecastHumidity;
