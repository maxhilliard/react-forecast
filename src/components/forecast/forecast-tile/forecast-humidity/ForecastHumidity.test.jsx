import React from 'react';
import { shallow } from 'enzyme';

import ForecastHumidity from './ForecastHumidity';

describe('ForecastHumidity', () => {
    describe('snapshot', () => {
        it('should match snapshot', () => {
            const wrapper = shallow(<ForecastHumidity humidity={54} />);

            expect(wrapper).toMatchSnapshot();
        });
    });
});
