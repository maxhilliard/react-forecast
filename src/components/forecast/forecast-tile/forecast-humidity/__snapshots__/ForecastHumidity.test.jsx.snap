// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastHumidity snapshot should match snapshot 1`] = `
<p
  className="forecastHumidity"
>
  Humidity:
  54
  %
</p>
`;
